src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,67 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import { ChangeEvent } from 'react';
+import { StatusFilter } from '../../helpers';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  handleStatus: (StatusFilter: StatusFilter) => void
+  handleQuery: (title: string) => void
+  query: string
+  clearQuery: () => void
+};
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+export const TodoFilter: React.FC<Props> = ({
+  handleStatus,
+  handleQuery,
+  clearQuery,
+  query,
+}) => {
+  const handleSelectStatus = (event: ChangeEvent<HTMLSelectElement>) => {
+    handleStatus(event.target.value as StatusFilter);
+  };
+
+  const handleSearchQuery = (event: ChangeEvent<HTMLInputElement>) => {
+    handleQuery(event.target.value);
+  };
+
+  return (
+    <form className="field has-addons">
+      <p className="control">
+        <span className="select">
+          <select
+            data-cy="statusSelect"
+            onChange={handleSelectStatus}
+          >
+            <option value={StatusFilter.ALL}>All</option>
+            <option value={StatusFilter.ACTIVE}>Active</option>
+            <option value={StatusFilter.COMPLETED}>Completed</option>
```

Comment:
You can render these options from your enum

```suggestion
            {Object.entries(FilterType).map(([label, value]) => (
              <option value={value} key={value}>
                {label}
              </option>
            ))}
```